go/cmd/vttestserver/main.go

```diff
@@ -165,6 +166,7 @@ func registerFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&config.ExternalTopoGlobalServerAddress, "external_topo_global_server_address", "", "the address of the global topology server for vtcombo process")
 	fs.StringVar(&config.ExternalTopoGlobalRoot, "external_topo_global_root", "", "the path of the global topology data in the global topology server for vtcombo process")
 
+	fs.DurationVar(&config.TopoTabletRefreshInterval, "tablet_refresh_interval", 10*time.Second, "Tablet refresh interval to Topo.")
```

Comment:
Any reason why we can't make this 5 seconds? Though of course people can override it to 5 using the provided env variable.

Reply:
yes I keep it conservative to 10s and if anyone else need to reduce the time more they can play around with it.

Reply:
I forget, is this an existing flag on vtgate that we are also adding to vttestserver?
New flags should use dashes not underscores (as the review checklist reminded me).

Reply:
Let's keep this as you have it, so that it is the same as the vtgate flag. No further change needed to this line.

Reply:
yes it is an already existing flag at vtgate level and we are just exposing it at vttestserver level.